src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx

```diff
@@ -131,27 +142,35 @@ We provide multiple ways to understand dependencies, so you can easily see how e
 
     <tr>
       <td>
-        Know which teams are responsible for specific entities across your organization.
+        Understand how everything is connected via API.
       </td>
 
       <td>
-        [Create tags](/docs/apis/graphql-api/tutorials/graphql-tagging-api-tutorial) for teams and all the services they monitor.
+        The [NerdGraph GraphiQL explorer](/docs/apis/graphql-api/getting-started/introduction-new-relic-graphql-api) manages all your entities, tags, and relationships.
       </td>
     </tr>
-  </tbody>
+    </tbody>
 </table>
 
 ## Query your data more easily [#chart-builder]
 
-At the top of all UI pages, **Query your data** is available. No matter your level of proficiency with our query language, you can create custom queries and charts:
+On the **Browse data** menu on the top navigation menu you can easily access your basic telemetry data (metrics, events, logs, and traces).
+
+Wherever you go in the UI, you can **Query your data** is available. No matter your level of proficiency with our query language, you can create custom queries and charts:
```

Comment:
I think that first sentence should either be:
"Wherever you go in the UI, **Query your data** is available."
or
"Wherever you go in the UI, you can **Query your data**."

Reply:
Thanks!